src/main/rx/rx.c

```diff
@@ -228,24 +238,37 @@ bool rxIsReceivingSignal(void)
     return rxSignalReceived;
 }
 
+bool rxAreFlightChannelsValid(void)
```

Comment:
Logical operation is not apparent from name - this test if at least one channel is valid (is this intended? Disconnected wire? )